src/evergreen/project.py

```diff
@@ -41,3 +43,25 @@ def __init__(self, json: Dict[str, Any], api: EvergreenApi) -> None:
     def __str__(self) -> str:
         """Get a string version of the Project."""
         return self.identifier
+
+    def most_recent_version(self) -> Version:
+        """
+        Fetch the most recent version.
+
+        :return: Version queried for.
+        """
+        version_iterator = self._api.versions_by_project(self.identifier)
+        return next(version_iterator)
+
+    def module_by_name(self, module_name: str) -> Optional[ManifestModule]:
```

Comment:
I wonder if this function would fit better on the `Version` object. Logically modules belong to a version, not to a project. Since different versions of the same project could have different sets of modules.

Also, in the manifest object, we could change the `modules` function to return an empty map instead of `None` if "modules" is not in the json. That would allow this to be simpler. I picture the usage like:
```
module = project.most_recent_version().modules.get(module_name)
```

Reply:
That makes sense, I think that would be better.